library/src/f3d_interactor.cxx

```diff
@@ -2,8 +2,8 @@
 
 namespace f3d
 {
-namespace details
-{
+
+// Is this ok to declare like this ? TODO
```

Comment:
the only things I'm not sure of in this MR @Meakk . This is declared in the cxx only.